R/validate_inputs.R

```diff
@@ -108,6 +108,11 @@ validate_inputs <- function(..., header = "Some inputs require attention") {
   lapply(vals, checkmate::assert_class, "InputValidator")
   checkmate::assert_string(header, null.ok = TRUE)
 
+  if (any(vapply(vals, function(iv) isFALSE(iv$.__enclos_env__$private$enabled), logical(1L)))) {
```

Comment:
Can we have a helper function to abstract out this

```
is_enabled <- function(iv) {
  isTRUE(iv$.__enclos_env__$private$enabled
} 
```
Then we can have

`if (!all(vapply(vals, is_enabled, logical(1))`  etc. and can simplify `Filter(is_enabled, vals)`

and if shinyvalidate change their implementation we only need to change in one place.

Reply:
wonder if it's worth adding an issue into  shinyvalidate to get `iv$is_enabled()` added for us...

Reply:
Done and done.